scheduler-jobs/deploy_data_pavics_jupyter.env

```diff
@@ -1,25 +1,90 @@
+# This will generates jobs for the "deploy_data_specific_image" script found on the birdhouse/pavics-jupyter-base image.
+# A job will be created for each image found in the DOCKER_NOTEBOOK_IMAGES variable,
+# defined in the env.local from birdhouse-deploy.
 #
-## Source deploy_data_image_specific_job.env that will actually perform the job creation.
-#. <path to deploy_data_image_specific_job.env>
+# These jobs assume the "scheduler" component is enabled in birdhouse-deploy.
 #
-## End Sample
-#############
+# Also, the image started by the job must contain the script "deploy_data_specific_image"
+# and a yaml config file "notebook_config.yml", and they must be placed on the root directory.
+# The script should be included in the pavics-jupyter-base image built from this repo.
+# Each specific images deriving from this base image should include its own "notebook_config.yml" file.
+##############################################################################
 
 DEPLOY_DATA_JOB_SCHEDULE="${AUTODEPLOY_NOTEBOOK_FREQUENCY}"
-DEPLOY_DATA_JOB_JOB_NAME="notebookdeploy-${DEPLOY_DATA_JOB_JUPYTERHUB_IMAGE_NAME}"
 DEPLOY_DATA_JOB_CONFIG="/notebook_config.yml"
-DEPLOY_DATA_JOB_JOB_DESCRIPTION="Auto-deploy tutorial notebooks for the image ${DEPLOY_DATA_JOB_DOCKER_IMAGE}"
 
+# Default path where notebooks are deployed on the image running the script
+NOTEBOOK_DEST_DIR=/tmp/notebooks
+
+# Cronjob schedule.
+if [ -z "$DEPLOY_DATA_JOB_SCHEDULE" ]; then
+    echo "Error: DEPLOY_DATA_JOB_SCHEDULE not set" 1>&2
+    exit 1
+    #DEPLOY_DATA_JOB_SCHEDULE="3 1,4,7,10,13,16,19,22 * * *"  # UTC
+fi
+
+for i in $( seq 1 $(echo $DOCKER_NOTEBOOK_IMAGES | wc -w)) ; do
+    DEPLOY_DATA_JOB_DOCKER_IMAGE=`echo $DOCKER_NOTEBOOK_IMAGES | cut -d " " -f $i`
+    DEPLOY_DATA_JOB_JUPYTERHUB_IMAGE_NAME=`echo $JUPYTERHUB_IMAGE_SELECTION_NAMES | cut -d " " -f $i`
+
+    DEPLOY_DATA_JOB_JOB_NAME="notebookdeploy-${DEPLOY_DATA_JOB_JUPYTERHUB_IMAGE_NAME}"
+    DEPLOY_DATA_JOB_JOB_DESCRIPTION="Auto-deploy tutorial notebooks for the image ${DEPLOY_DATA_JOB_DOCKER_IMAGE}"
+
+    # Docker image to run deploy-date script.
+    if [ -z "$DEPLOY_DATA_JOB_DOCKER_IMAGE" ]; then
+        echo "Error: DEPLOY_DATA_JOB_DOCKER_IMAGE not set" 1>&2
+        exit 1
+    fi
+
+    # Name of the image found on the JupyterHub selection page, important for the path of the volume mount
+    if [ -z "$DEPLOY_DATA_JOB_JUPYTERHUB_IMAGE_NAME" ]; then
+        echo "Error: DEPLOY_DATA_JOB_JUPYTERHUB_IMAGE_NAME not set" 1>&2
+        exit 1
+    fi
+
+    # Log file location.  Default location under /var/log/PAVICS/ has built-in logrotate.
+    if [ -z "$DEPLOY_DATA_JOB_LOGFILE" ]; then
+        DEPLOY_DATA_JOB_LOGFILE="${PAVICS_LOG_DIR}/${DEPLOY_DATA_JOB_JOB_NAME}.log"
+    fi
+
+    ##############################################################################
+    # End configuration vars
+    ##############################################################################
```

Comment:
LOL, there's not start so might as well remove this "End configuration vars".

Reply:
Remains of the old copied code :P I will remove it!